lib/buffy/throttle.ex

```diff
@@ -219,6 +223,17 @@ defmodule Buffy.Throttle do
       @impl GenServer
       @spec handle_info(:timeout, Buffy.Throttle.state()) :: {:stop, :normal, Buffy.Throttle.state()}
       def handle_info(:timeout, {key, args}) do
+        selected_jitter = 1 - :rand.uniform(unquote(jitter) + 1)
```

Comment:
The docstring says default is set to 0, but that default in `:rand.uniform(0 + 1)` coulud lead to 0 or 1

Reply:
Ah, good find. Updated to wrap this in `min(result, 0)` to avoid `Process.sleep(-1)` which would cause an error.